src/plugins/formulas/__tests__/formulas.spec.js

```diff
@@ -386,93 +392,66 @@ describe('Formulas general', () => {
       height: 300
     });
 
-    expect(hot.getDataAtCell(0, 1)).toBe('#REF!');
+    expect(hot.getDataAtCell(0, 1)).toBe('#CYCLE!');
   });
 
-  it('should mark cell as #REF! (out of data table range for columns)', () => {
-    const hot = handsontable({
-      data: getDataForFormulas(0, 'name', ['=K1']),
-      columns: getColumnsForFormulas(),
-      formulas: true,
-      width: 500,
-      height: 300
+  // Discussion on why `null` instead of `#REF!` at
+  // https://github.com/handsontable/handsontable/issues/7668
+  describe('out of range cells should return null', () => {
+    it('columns', () => {
+      const hot = handsontable({
+        data: getDataForFormulas(0, 'name', ['=K1']),
+        columns: getColumnsForFormulas(),
+        formulas: true,
+        width: 500,
+        height: 300
+      });
+
+      expect(hot.getDataAtCell(0, 1)).toBe(null);
```

Comment:
This test (and the analogous one for rows) fails, when `evaluateNullToZero: true` is set for HF. Should it be enabled, @wojciechczerniak? If so, this case should probably be modified.

Reply:
We should probably just all sit down together and go through every single config option in HF (as discussed some time ago) to avoid ambiguities like this in the future.

Reply:
👍 True. We should do it in #7823

Reply:
Done in #7823, we can close this discussion here